src/scripts/run_certbot.sh

```diff
@@ -49,12 +49,39 @@ fi
 # $1: The name of the certificate (e.g. domain-rsa)
 # $2: String with all requested domains (e.g. -d domain.org -d www.domain.org)
 # $3: Type of key algorithm to use (rsa or ecdsa)
+# $4: The authenticator to use to solve the challenge
 get_certificate() {
-    info "Requesting an ${3^^} certificate for '${1}'"
+    local authenticator_params=
+    local challenge_type=
+
+    if [[ "$4" == dns-* ]] && \
+            [[ ${CERTBOT_DNS_AUTHENTICATORS} =~ (^| )${4#dns-}( |$) ]]; then
+        local provider="${4#dns-}"
+        local configfile="/etc/letsencrypt/${provider}.ini"
+        if [ ! -f "$configfile" ]; then
+            error "Authenticator is ${4} but ${configfile} is missing"
```

Comment:
I would like single quotes around the variables so it will be easier to identify them when reading the logs later.